src/ResourceManager/Network/Commands.Network.Test/Commands.Network.Test.csproj

```diff
@@ -104,6 +105,9 @@
     <None Include="ScenarioTests\ApplicationGatewayTests.ps1">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
+    <None Include="ScenarioTests\AvailableDelegationsTests.ps1">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </None>
```

Comment:
looks like you are copying over the tests but not the recordings (see line 210 of this file for example)

Reply:
thanks